modules/fluid_properties/doc/content/source/userobjects/TabulatedFluidProperties.md

```diff
@@ -21,17 +18,22 @@ Using an interpolation of tabulated fluid properties can significantly reduce th
 for computing fluid properties defined using complex equations of state, which may reduce the overall
 computational cost dramatically, especially if fluid properties are calculated a large number of times.
 
+!alert note
+`TabulatedFluidProperties` is a base class and may not be used. A derived class, specifying an interpolation
+method to use to interpolate between the tabulated data, must be used instead.
+Currently only [bicubic tabular interpolation](TabulatedBicubicFluidProperties.md) is implemented.
+Bilinear interpolation is a work in progress.
+
 ## File format
 
 The expected file format for the tabulated fluid properties is now described.  The first line must be
 the header containing the required column names *pressure* and *temperature*, and also any number of
-the fluid properties that TabulatedFluidProperties understands. These are: *density*, *enthalpy*,
-*internal_energy*, *viscosity*, *k* (the thermal conductivity), *cp* (the isobaric specific heat
-capacity), *cv* (the isochoric specific heat capacity), and *entropy*.
+the fluid properties that TabulatedFluidProperties understands. These are: *density*, *enthalpy*, *v* (specific_volume),
+*internal_energy*, *viscosity*, *k* (the thermal conductivity), g (gibbs free energy), *cp* (the isobaric specific heat
+capacity), *cv* (the isochoric specific heat capacity), *c* (speed of sound in fluid), and *entropy*.
 
 !alert note
-The order is not important, although having pressure and temperature first makes the data easier
-for a human to read.
+The order is not important, although having pressure and temperature first makes the data easier for a human to read.
 
 The data in the pressure and temperature columns *must* be monotonically increasing. This file format
 does require duplication of the pressure and temperature data - each pressure value must be included
```

Comment:
Not your changes, but it'd be nice to use symbols instead of "num_T" and "num_p"